<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_title</class>
 <widget class="QDialog" name="dlg_title">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>259</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Authentication</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;Ubuntu Condensed&quot;;</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QCheckBox" name="chb_memory">
     <property name="toolTip">
      <string>Do not ask for my credentials for the next 30 minutes</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Ubuntu Condensed&quot;;
color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>Remember me</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="lbl_mdpTitle">
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Ubuntu Condensed&quot;;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCommandLinkButton" name="clb_forgot">
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Ubuntu Condensed&quot;;
font: italic 11pt &quot;Ubuntu Condensed&quot;;
color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>Forgotten password</string>
     </property>
     <property name="icon">
      <iconset theme="link">
       <normaloff>../toolbelt</normaloff>../toolbelt</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QPushButton" name="btn_connection">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);
font: 75 15pt &quot;Ubuntu Condensed&quot;;</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QgsPasswordLineEdit" name="ple_mdpEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="lbl_userTitle">
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Ubuntu Condensed&quot;;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
     <property name="text">
      <string>Username or email</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QCommandLinkButton" name="clb_url">
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;Ubuntu Condensed&quot;;
font: italic 11pt &quot;Ubuntu Condensed&quot;;
color: rgb(52, 101, 164);</string>
     </property>
     <property name="text">
      <string>New here? Don't have an account yet? Click me to sign-in!</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLineEdit" name="lbl_userEdit">
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>firstname.name@gis-is-what-i-do.com</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lbl_userEdit</tabstop>
  <tabstop>ple_mdpEdit</tabstop>
  <tabstop>btn_connection</tabstop>
  <tabstop>chb_memory</tabstop>
  <tabstop>clb_forgot</tabstop>
  <tabstop>clb_url</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
